src/components/Button/Button.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
 export default function Button(props) {
-  const { type, text } = props
+  const { type = 'button', text, children } = props
```

Comment:
```suggestion
  const { type = 'button', children } = props
```
children thì đâu cần text làm gì nữa đâu em
Lúc này nó đã là như này rồi mà
`<Button>Login</Button>`
hoặc
```
<Button>
   <Icon icon='...' />
   Login
</Button>
```

Reply:
Em đã fix